js/apps/account-ui/src/utils/useFormatDate.ts

```diff
@@ -0,0 +1,16 @@
+const DATE_AND_TIME_FORMAT: Intl.DateTimeFormatOptions = {
+  dateStyle: "long",
+  timeStyle: "short",
+};
+
+//todo use user local
+export default function useFormatDate() {
+  return {
+    formatDate: function (
+      date: Date,
+      options: Intl.DateTimeFormatOptions | undefined = DATE_AND_TIME_FORMAT,
```

Comment:
When providing a default value the parameter is already considered optional, so there should be need to to add the `undefined` type here.
```suggestion
      options: Intl.DateTimeFormatOptions = DATE_AND_TIME_FORMAT,
```